content/docs/intro/concepts/stack.md

```diff
@@ -28,9 +30,13 @@ If you are using Pulumi in your organization, by default the stack will be creat
 $ pulumi stack init broomllc/staging
 ```
 
-Fully qualified stack names also include the project name, in the form `orgName/projectName/stackName`, and this fully-qualified format is required in some contexts.  In most contexts, the shorthands `orgName/stackName` or `stackName` are valid and use the default organization and the current project context.
+Fully qualified stack names also include the project name, in the form `orgName/projectName/stackName`, and this fully-qualified format is required in some contexts. In most contexts, the shorthands `orgName/stackName` or `stackName` are valid and use the default organization and the current project context.
```

Comment:
`in some contexts` ... such as?

Also, what is "the default organization"? Do you mean my username? I don't think we have the notion of a default organization (other than the user's personal organization), so this is probably worth a bit more clarification.

Reply:
Will address this in a follow-up issue.